### ISSUE TITLE
Support for cjsx/coffee-react

### ISSUE THREAD

**Issue by TrySpace** [jashkenas/docco#354]
Im using `gulp-cjsx`/`coffee-react` to transpile coffeescript jsx files (cjsx)

But using docco with these will throw an error `docco: skipped unknown type` which doesn't surprise me.

```
    fs.js:491
      binding.open(pathModule._makeLong(path),
              ^
   TypeError: path must be a string
       at TypeError (native)
       at Object.fs.open (fs.js:491:11)
       at fs.readFile (fs.js:262:6)
```

Now I don't know much about transpiling, and how much work it would be to add support for `jsx` tags, which are basically `html` tags inside your script (I takes some getting used to, but sublime text has working syntax highliters for this)

But I doubt you'd have to write a lot of new stuff for this, if you can use the above mentioned modules, it might be just a matter of detecting/adding the `cjsx` filetype and directing it to interpret it differently. Or, just altogether ignore all jsx syntax, because it's not needed for documentations.

**Comment by TrySpace** [comment]
I've temporarily fixed it by adding this to `language.json`:

      .cjsx":        {"name": "coffeescript", "symbol": "#"},

That works, but would be nice if it came with it.

**Comment by jashkenas** [comment]
Merged #355
